docs/api/basic.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Core
+title: Basic
```

Comment:
Can we keep core.mdx, and have two sections "Basic APIs" and "Additional APIs" in it?

Reply:
Sure, done.